examples/hello_world_different_sizes.rs

```diff
@@ -0,0 +1,123 @@
+use std::{io::stdout, thread::sleep, time::Duration};
+
+use anyhow::Result;
+use crossterm::{
+    terminal::{disable_raw_mode, enable_raw_mode, EnterAlternateScreen, LeaveAlternateScreen},
+    ExecutableCommand,
+};
+use ratatui::{
+    prelude::*,
+    widgets::{Block, Borders},
+};
+use tui_big_text::BigTextBuilder;
+
+fn main() -> Result<()> {
+    stdout().execute(EnterAlternateScreen)?;
+    enable_raw_mode()?;
+    let backend = CrosstermBackend::new(stdout());
+    let mut terminal = Terminal::new(backend)?;
+    terminal.clear()?;
+    terminal.draw(|frame| render(frame).expect("failed to render"))?;
+    sleep(Duration::from_secs(25));
+    terminal.clear()?;
+    stdout().execute(LeaveAlternateScreen)?;
+    disable_raw_mode()?;
+    Ok(())
+}
+
+fn render(frame: &mut Frame) -> Result<()> {
+    // Setup layout for 4 blocks
+    let outer_layout = Layout::default()
+        .direction(Direction::Vertical)
+        .constraints([Constraint::Percentage(50), Constraint::Percentage(50)].as_ref())
+        .split(frame.size());
```

Comment:
This can be more succinctly:
```suggestion
    let outer_layout = Layout::new(Direction::Vertical), [Constraint::Ratio(1,2); 2])
        .split(frame.size());
```

Also (just FYI), in the next version of ratatui, this will collapse to:

```suggestion
    let [top, bottom] = frame.size().split(Layout::vertical([Constraint::Ratio(1,2); 2]));
    let [top_left, top_right[ = top.split(Layout::horizontal([Constraint::Ratio(1,2);
    let [bottom_left, bottom_right[ = bottom.split(Layout::horizontal([Constraint::Ratio(1,2); 2]));
```

Reply:
nice thing, the new layout handling in the second suggestion for the next release looks a lot more intuitive.

Reply:
For now I just refactored the layouts as in the first suggestion